elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -0,0 +1,113 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: clipboard;
+
+// Example local variable, using BEM format
+// --pfe-clipboard__region--Property: value;
+
+// This variable is global so that helper functions can reference it
+$LOCAL-VARIABLES: (
+  FontSize: 1rem,
+  FontWeight: pfe-var(font-weight--light),
+  Color: pfe-broadcasted(link),
+  Color--hover: pfe-broadcasted(link--hover),
+  Color--focus: pfe-broadcasted(link--focus),
+  text--success--Color: pfe-var(feedback--success),
+  icon--Width: 16px,
+  icon--Height: auto,
+);
+
+:host {
+  display: flex;
+  flex: 0 0 auto;
+  align-items: center;
+  max-width: fit-content;
+  color: pfe-local(Color) !important;
+  font-weight: pfe-local(FontWeight);
+  cursor: pointer;
+  // @todo: How should we handle padding with inline elements?
```

Comment:
It looks like you switched it to `display: flex`, is this question still relevant?

Reply:
bump ^